examples/yew/index.html

```diff
@@ -1,19 +1,23 @@
 <!DOCTYPE html>
 <html>
-<head>
-    <meta charset="utf-8"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>Trunk | Yew | YBC</title>
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css"/>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css"
+    />
 
-    <link data-trunk rel="scss" href="src/index.scss"/>
-    <link data-trunk rel="css" href="src/app.css"/>
-    <link data-trunk rel="icon" href="src/yew.svg"/>
-    <link data-trunk rel="copy-file" href="src/yew.svg"/>
-    <link data-trunk rel="inline" type="css" href="src/inline-css.css"/>
-    <base data-trunk-public-url/>
-</head>
-<body>
-    <link data-trunk rel="rust" href="Cargo.toml" data-bin="yew-example"/>
-</body>
+    <link data-trunk rel="scss" href="src/index.scss" />
+    <link data-trunk rel="scss" href="src/inline-scss.scss" data-inline />
+    <link data-trunk rel="css" href="src/app.css" />
+    <link data-trunk rel="icon" href="src/yew.svg" />
+    <link data-trunk rel="copy-file" href="src/yew.svg" />
+    <link data-trunk rel="inline" type="css" href="src/inline-css.css" />
+    <base data-trunk-public-url />
+  </head>
+  <body>
+    <link data-trunk rel="rust" href="Cargo.toml" data-bin="yew-example" />
+  </body>
```

Comment:
Let's revert any changes which are not actually part of the primary focus of this PR. Looks like there is an inline scss piece added, that's fine. Please match the current style of the HTML for that new element.